lib/NetworkManager.js

```diff
@@ -116,7 +116,7 @@ class NetworkManager extends EventEmitter {
         response = 'ProvideCredentials';
         this._attemptedAuthentications.add(event.interceptionId);
       }
-      const {username, password} = this._credentials || {};
+      const {username, password} = this._credentials || {username: undefined, password: undefined};
```

Comment:
is this necessary now?

Reply:
Yes. I think it will help more than hurt.